pkg/adaptation/api.go

```diff
@@ -102,7 +103,8 @@ type (
 )
 
 // Aliased consts for api/api.proto.
-// nolint
+//
+//nolint:all
```

Comment:
This one we need to keep as a selective one for `revive`... something like `//nolint:revive // [ignore warnings about generated types, names, etc.]`